projects/panoviewer.md

```diff
@@ -0,0 +1,16 @@
+---
+abstract: JavaScript library for 360° street views of Amsterdam, Almere and Amstelveen. 
+---
+
+# PanoViewer
+
+[![PanoViewer Demo](../images/panoviewer-amsterdam.png)](https://panoviewer.data.amsterdam.nl/demo/)
+
+A javascript library to display interactive street view 360° panorama images of Amsterdam, Almere and Amstelveen and navigate between them.
```

Comment:
`interactive 360° street view panorama images`? Lijkt me logischer, en meer in lijn met de rest van de tekst.